Dockerfile

```diff
@@ -1,51 +1,77 @@
 #FROM nvcr.io/nvidia/pytorch:20.06-py3
 #FROM continuumio/miniconda3
 #FROM gpuci/miniconda-cuda:11.0-devel-ubuntu20.04
-FROM nvidia/cuda:11.1-cudnn8-devel-ubuntu20.04
-
-#COPY environment.yml .
-#RUN conda update --force-reinstall conda
-#RUN conda env update --name base --file environment.yml --prune
+FROM nvidia/cuda:11.1-cudnn8-devel-ubuntu20.04 as BUILD
 
 ENV MY_ROOT=/workspace \
     PKG_PATH=/yolo_src \
     NUMPROC=4 \
     PYTHON_VER=3.8 \
     PYTHONUNBUFFERED=1 \
     PYTHONPATH=. \
-    DEBIAN_FRONTEND=noninteractive 
+    DEBIAN_FRONTEND=noninteractive
 
 WORKDIR $PKG_PATH
 
-RUN apt-get update && apt-get install -y apt-utils && apt-get -y upgrade && \
+RUN apt-get update && apt-get install -y apt-utils && apt-get -y dist-upgrade && \
     apt-get install -y git libsnappy-dev libopencv-dev libhdf5-serial-dev libboost-all-dev libatlas-base-dev \
         libgflags-dev libgoogle-glog-dev liblmdb-dev curl unzip\
         python${PYTHON_VER}-dev && \
     curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py && \
     python${PYTHON_VER} get-pip.py && \
     rm get-pip.py && \
     # Clean UP
-    apt upgrade -y && \
-    apt clean && \
-    apt autoremove -y && \
+    apt=get autoremove -y && \
+    apt-get autoclean -y && \
     rm -rf /var/lib/apt/lists/*  # cleanup to reduce image size
 
 RUN ln -s /usr/bin/python${PYTHON_VER} /usr/bin/python
 
-RUN apt install unzip
-RUN pip install torch==1.7.1+cu110 torchvision==0.8.2+cu110 -f https://download.pytorch.org/whl/torch_stable.html
+RUN pip install --no-cache-dir torch==1.7.1+cu110 torchvision==0.8.2+cu110 -f https://download.pytorch.org/whl/torch_stable.html
 
 WORKDIR $MY_ROOT
 # We have to install mish-cuda from source due to an issue with one of the header files
 ADD https://github.com/thomasbrandon/mish-cuda/archive/master.zip $MY_ROOT/mish-cuda.zip
```

Comment:
Not sure if this will be any easier to install but the updated README suggests to use this repo https://github.com/JunnYu/mish-cuda instead, I wonder if you could then just pip install it and avoid the devel files altogether.